PhysicsTools/NanoAOD/python/custom_jme_cff.py

```diff
@@ -25,7 +25,8 @@
   'pfDeepFlavourJetTags:probc','pfDeepFlavourJetTags:probuds','pfDeepFlavourJetTags:probg'
 ]
 from RecoBTag.ONNXRuntime.pfParticleNetAK4_cff import _pfParticleNetAK4JetTagsAll
-bTagDiscriminatorsForAK4 = cms.PSet(foo = cms.vstring(bTagDeepCSV+bTagDeepJet+_pfParticleNetAK4JetTagsAll))
+from RecoBTag.ONNXRuntime.pfParticleTransformerAK4_cff import _pfParticleTransformerAK4JetTagsAll
+bTagDiscriminatorsForAK4 = cms.PSet(foo = cms.vstring(bTagDeepCSV+bTagDeepJet+_pfParticleNetAK4JetTagsAll+_pfParticleTransformerAK4JetTagsAll))
```

Comment:
Ah, right I can drop it here as well - for the jme_cff I tried to introduce only minimal changes (additions, basically), as I was not fully sure what variables are still interesting to monitor. But as we will remove it also earlier in the chain this would result in problems downstream. Thanks for spotting it! On my agenda for the next commit.

Reply:
sorry, you'r right. I though this was in nano_cff. We can leave it there for now and then check with JME what they prefer to do with it.